src/airflow/dags/airqo_measurements.py

```diff
@@ -483,8 +483,58 @@ def update_latest_data_topic(data: pd.DataFrame):
     update_latest_data_topic(calibrated_data)
 
 
+@dag(
+    "AirQo-Raw-Data-Low-Cost-Measurements",
+    schedule="*/5 * * * *",
+    default_args=AirflowUtils.dag_default_configs(),
+    catchup=False,
+    tags=["airqo", "raw", "low cost"],
+)
+def airqo_raw_data_measurements():
+    import pandas as pd
+    from airqo_etl_utils.date import date_to_str_hours
+    from datetime import datetime, timedelta
+
+    hour_of_day = datetime.utcnow() - timedelta(hours=1)
+    start_date_time = date_to_str_hours(hour_of_day)
+    end_date_time = datetime.strftime(hour_of_day, "%Y-%m-%dT%H:59:59Z")
```

Comment:
Thanks @BenjaminSsempala 
This can be transferred to `extract_raw_data` function and interval reduced to 30 minutes

Reply:
Done @NoahNsimbe